native-code/ios/index.md

```diff
@@ -9,41 +9,49 @@ crumb: iOS
 {% include toc-hide.html %}
 
 
-### Development environment
+### Development Environment
 
 An OS X machine is required for iOS development. While it's possible to
 develop purely from the command line with text editors, it's easiest to use
-XCode. Both methods will be illustrated here.
+Xcode. Both methods will be illustrated here.
 
 
-### Getting the code
+### Getting the Code
 
   1. Install [prerequisite software][1]
 
   2. Set the target OS in your environment:
 
      ~~~~~ bash
-     export GYP_DEFINES="OS=ios"
+     export GYP_CROSSCOMPILE=1
+     export GYP_DEFINES="OS=ios target_arch=arm"
+     export GYP_GENERATOR_FLAGS="output_dir=out_ios"
+     export GYP_GENERATORS=ninja
      ~~~~~
 
   3. Create a working directory, enter it, and run:
 
      ~~~~~ bash
-     fetch webrtc_ios
+     fetch --nohooks webrtc_ios
+     gclient sync
      ~~~~~
 
+     This will fetch a regular WebRTC checkout with the iOS-specific parts
+     added. The same checkout can be used for both Mac and iOS development,
+     depending on the OS you set in `GYP_DEFINES` (see above).
+
   4. You may want to disable Spotlight indexing for the checkout to speed up
-     file operations.
+     file operations..
 
 See [Development][2] for generic instructions on how
 to update the code in your checkout.
 
 
-### Compiling the code
+### Compiling the Code
 
 GYP is used to generate build instructions for ninja from the relevant .gyp files. Ninja is used to compile the source using the previously generated instructions. In order to configure GYP to generate build files for iOS certain environment variables need to be set. Those variables can be edited for the various build configurations as needed.
 
-Building for iOS device:
+Building for iOS Device:
 
 ~~~~~ bash
 export GYP_CROSSCOMPILE=1
```

Comment:
You can remove this line, it's not needed (but it was present on https://sites.google.com/site/webrtc/native-code/ios)